packages/admin/cms-admin/src/pages/pageTree/PageContextMenu.tsx

```diff
@@ -100,7 +100,7 @@ const PageContextMenu = (props: PageContextMenuProps): React.ReactElement => {
                         <ListItemIcon>
                             <Edit />
                         </ListItemIcon>
-                        <ListItemText primary={<FormattedMessage {...messages.content} />} />
+                        <ListItemText primary={intl.formatMessage({ id: "comet.generic.editContent", defaultMessage: "Edit content" })} />
```

Comment:
```suggestion
                        <ListItemText primary={intl.formatMessage({ id: "comet.pages.pages.page.editContent", defaultMessage: "Edit content" })} />
```

This is not a generic message 😉